aiofreepybox/api/home.py

```diff
@@ -0,0 +1,149 @@
+# Home structure : adapter > node > endpoint
+class Home:
+
+    def __init__(self, access):
+        self._access = access
+
+    async def get_adapter(self, adapterid):
+        '''
+        Retrieve a registered HomeAdapter
+        '''
+        return await self._access.get('home/adapters/', adapterid)
```

Comment:
The `get` method takes a single parameter, I think this should be `get('home/adapters/{0}'.format(adapterid)`.

Reply:
fixed